wgsl_bindgen/src/wgsl_bindgen.rs

```diff
@@ -210,14 +213,15 @@ impl WGSLBindgen {
   }
 
   fn generate_naga_module_for_entry(
+    capabilities: crate::Capabilities,
     entry: SourceWithFullDependenciesResult<'_>,
   ) -> Result<WgslEntryResult, WgslBindgenError> {
     let map_err = |err: ComposerError| WgslBindgenError::NagaModuleComposeError {
       entry: entry.source_file.file_path.to_string(),
       inner: err.inner,
     };
 
-    let mut composer = Composer::default();
+    let mut composer = Composer::default().with_capabilities(capabilities);
```

Comment:
Would update it to work with Option, such that `with_capabilities(capabilities)` isn't needed when it's `None` by default

Reply:
use `Option` and `match` now.